py/stream.c

```diff
@@ -33,7 +33,7 @@
 #include "py/runtime.h"
 
 #if MICROPY_STREAMS_NON_BLOCK
-#include <errno.h>
+#include "py/mperrno.h"
 #if defined(__MINGW32__) && !defined(__MINGW64_VERSION_MAJOR)
 #define EWOULDBLOCK 140
 #endif
```

Comment:
I think this entire block can now be removed because EWOULDBLOCK is never needed, and mperrno.h is included by stream.h.

Reply:
You're right, it's not needed. I've removed the entire block.